assets/src/scripts/index.js

```diff
@@ -31,11 +31,13 @@ function main() {
 
     } catch (err) {
         // Send exception to Google Analytics.
-        window.ga('send', 'exception', {
-            // exDescription will be truncated at 150 bytes
-            exDescription: err.stack,
-            exFatal: true
-        });
+        if (window.ga) {
```

Comment:
😆 I love it when exception handling code raises its own exceptions!

Reply:
Yeah. I had this happen more than once before this, but with the scope of changes here, I really needed my exception traces!